src/format/command.rs

```diff
@@ -97,13 +118,14 @@ fn format_file(path: PathBuf) -> Result<Formatted, Error> {
     })?;
 
     Ok(Formatted {
-        path,
+        source: path.clone(),
+        destination: FormatDestination::File { path: path },
         input: src,
         output: formatted,
     })
 }
 
-pub fn format_stdin(_check: bool) -> Result<(), Error> {
+pub fn read_and_format_stdin() -> Result<Vec<Formatted>, Error> {
```

Comment:
Perhaps it'd be better to just return `Result<Formatted, Error>` and then wrap it in `vec!` at the callsite?

Reply:
I think that's a good idea :)